cmd/e2e-runner/main.go

```diff
@@ -63,9 +63,12 @@ func main() {
 	logger.Info("successful shutdown")
 }
 
-func randomString() string {
-	rand.Seed(time.Now().Unix())
-	return fmt.Sprintf("%x", rand.Int63())
+func randomString() (string, error) {
+	n, err := rand.Int(rand.Reader, big.NewInt(10000))
```

Comment:
It's probably better to read some number of bytes (e.g. 16), then base64 or hex-encode the result. We do this in a few test helpers.